packages/shared/src/components/auth/OnboardingSteps/ContentTypes/ContentTypes.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { ReactElement } from 'react';
+import useFeedSettings from '../../../../hooks/useFeedSettings';
+
+import { useAdvancedSettings } from '../../../../hooks';
+import { CustomCheckbox } from './CustomCheckbox';
+import { useContentTypesOnboarding } from './useContentTypesOnboarding';
+
+export const ContentTypes = (): ReactElement => {
+  const { contentSourceList } = useFeedSettings();
+  const {
+    selectedSettings,
+    onToggleSettings,
+    checkSourceBlocked,
+    onToggleSource,
+  } = useAdvancedSettings();
+  const { advancedSettingsExceptCommunity } = useContentTypesOnboarding();
+
+  return (
+    <div className="flex max-w-[63.75rem] flex-col tablet:px-10">
```

Comment:
Where does this max-w come from? feels a bit odd

Reply:
So the box does not expand more then it should in big screens. it is a set width in figma

Reply:
I think we can use max-w-laptop then right? (Or something like that)

Reply:
oh yeah true